es/lecciones/intro-a-google-maps-y-google-earth.md

```diff
@@ -0,0 +1,562 @@
+    genera un archivo KML.
+
+-   Para compartir archivos KML simplemente ubica el archivo que guardaste en tu
+    computadora y súbelo a tu sitio web, tu perfil de redes sociales o envíalo
+    como adjunto en un correo electrónico.
+
+**Ya aprendiste a utilizar Google Maps y Google Earth. ¡Asegúrate de guardar tu
+trabajo!**
+
+*Esta lección es parte de [Geospatial Historian](https://geospatialhistorian.wordpress.com/)*
+
+  [Google Maps Engine Lite]: https://mapsengine.google.com
+  [geo-es1]: /images/intro-a-google-maps-y-google-earth/geo-es1.png
+  [geo-es2]: /images/intro-a-google-maps-y-google-earth/geo-es2.png
+  [geo-es3]: /images/intro-a-google-maps-y-google-earth/geo-es3.png
+  [geo-es4]: /images/intro-a-google-maps-y-google-earth/geo-es4.png
+  [geo-es5]: /images/intro-a-google-maps-y-google-earth/geo-es5.png
+  [geo-es6]: /images/intro-a-google-maps-y-google-earth/geo-es6.png
+  [Archivo CSV del Suministro global de grasa de Reino Unido]: /assets/Suministro_global_de_grasa_de_Reino_Unido_1894_1896.zip
+  [geo-es7]: /images/intro-a-google-maps-y-google-earth/geo-es7.png
+  [geo-es8]: /images/intro-a-google-maps-y-google-earth/geo-es8.png
+  [geo-es9]: /images/intro-a-google-maps-y-google-earth/geo-es9.png
+  [geo-es10]: /images/intro-a-google-maps-y-google-earth/geo-es10.png
+  [geo-es11]: /images/intro-a-google-maps-y-google-earth/geo-es11.png
+  [geo-es12]: /images/intro-a-google-maps-y-google-earth/geo-es12.png
+  [Creating New Vector Layers in QGIS 2.0]: /lessons/vector-layers-qgis
+  [geo-es13]: /images/intro-a-google-maps-y-google-earth/geo-es13.png
+  [geo-es14]: /images/intro-a-google-maps-y-google-earth/geo-es14.png
+  [geo-es15]: /images/intro-a-google-maps-y-google-earth/geo-es15.png
+  [geo-es16]: /images/intro-a-google-maps-y-google-earth/geo-es16.png
+  [geo-es17]: /images/intro-a-google-maps-y-google-earth/geo-es17.png
+  [geo-es18]: /images/intro-a-google-maps-y-google-earth/geo-es18.png
+  [geo-es19]: /images/intro-a-google-maps-y-google-earth/geo-es19.png
+  [geo-es20]: /images/intro-a-google-maps-y-google-earth/geo-es20.png
+  [geo-es21]: /images/intro-a-google-maps-y-google-earth/geo-es21.png
+  [geo-es22]: /images/intro-a-google-maps-y-google-earth/geo-es22.png
+  [geo-es23]: /images/intro-a-google-maps-y-google-earth/geo-es23.png
+  [geo-es24]: /images/intro-a-google-maps-y-google-earth/geo-es24.png
+  [www.davidrumsey.com]: http://www.davidrumsey.com/
+  [geo-es25]: /images/intro-a-google-maps-y-google-earth/geo-es25.png
+  [geo-es26]: /images/intro-a-google-maps-y-google-earth/geo-es26.png
+  [Georeferencing in QGIS 2.0]: /lessons/georeferencing-qgis
+  [geo-es27]: /images/intro-a-google-maps-y-google-earth/geo-es27.png
+  [geo-es28]: /images/intro-a-google-maps-y-google-earth/geo-es28.png
+  [Maps Engine Lite]: https://mapsengine.google.com/map/
+  [mapa de la vía marítima]: https://github.com/programminghistorian/jekyll/files/148993/seaway.zip
```

Comment:
I don't see a file named `seaway.zip` in the actual git repository. Was this uploaded to a github issue? Please copy this file to the `/assets` directory and use the link `/assets/sewaway.zip` (no `https://programminghistorian.org` before it.)

**Before you upload the file, remember to change the branch to `mariajoafana-patch-1` so that the change is added to this repository.**

Reply:
Done, I couldn't find the file in Jekyll, so I downloaded it and uploaded it to /assets  I tried branching it to mariajoafana-patch-1 but it changed the path to patch-1-1 . Thanks for the patience, I'm still trying to understand how all this works...

Reply:
@mariajoafana make sure to follow the instructions here https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#the-additional-change 

Change to the `mariajoafana-patch-1` branch _before_ editing or uploading files. If you do that, you can make commits without opening new pull requests. If GitHub asks you to open a new pull request, then you've tried to commit to the `gh-pages` branch accidentally. You'll need to go back and do it again.

Reply:
I think that a training session would be a good idea. I already went through the instructions, things are more clear now, thanks @mdlincoln 
I just uploaded the file seaway.zip making sure that I changed to mariajoafana-patch-1 before uploading the file